src/lib/telldus-accessory.js

```diff
@@ -46,6 +46,8 @@ class TelldusAccessory {
     this.model = modelPair[0]
     this.manufacturer = modelPair[1]
 
+    this.dimlevel = 0
```

Comment:
I don't believe this is the correct place for it. Saying that a switch, or a thermometer has a dim level does not make sense. I'd prefer this in a constructor for the TelldusDimmer instead. You can add it as such:
```js
  /**
   * Telldus Dimmer constructor.
   * 
   * We need dimmers to keep track of the dim level because...
   */
  constructor(data, log, homebridge, config) {
    super(data, log, homebridge, config)
    this.dimlevel = 0
  }
```